assets/src/dashboard/components/layout/provider.js

```diff
@@ -48,47 +48,46 @@ export const LayoutContext = createContext(null);
 const Provider = ({ children }) => {
   const [squishContentHeight, setSquishContentHeight] = useState(0);
   const scrollFrameRef = useRef(null);
-  const squishDriverRef = useRef(null);
 
   useLayoutEffect(() => {
-    const squishDriverEl = squishDriverRef.current;
     const scrollFrameEl = scrollFrameRef.current;
-    if (!(squishDriverEl || scrollFrameEl)) {
+    if (!scrollFrameEl) {
       return () => {};
     }
 
-    const observer = new IntersectionObserver(
-      (entries) => {
-        entries.forEach((entry) => {
-          if (entry.intersectionRatio) {
-            dispatchSquishEvent(squishDriverEl, 1 - entry.intersectionRatio);
-          }
-        });
-      },
-      {
-        root: scrollFrameEl,
-        threshold: generateThreshold(1000),
+    let prevProgress;
+    const handleScroll = throttleToAnimationFrame((e) => {
```

Comment:
nit: if `target` is the only thing getting used out of `e` might as well destructure it `(({target}) => `